app/src/app/pages/plando-page/plando-items/plando-items.component.ts

```diff
@@ -47,6 +48,7 @@ export class PlandoItemsComponent {
   public massFillCheckTypes = Object.values(CheckType).filter(val => !this.filteredTypes.includes(val) && val !== CheckType.NORMAL);
   public filteredItems: string[] = this.PLANDO_ITEMS.slice();
   public searchText: FormControl;
+  public settingTrap: boolean = false;
```

Comment:
Looks like this isn't actually used aside from assignations?

Reply:
Whoops! Good catch. Fixed.